core/sql/ustat/hs_globals.cpp

```diff
@@ -4392,6 +4388,12 @@ Lng32 HSSample::make(NABoolean rowCountIsEstimate, // input
         LM->Log(LM->msg);
       }
 
+ 
+    double sampleRatio = samplePercent / 100;
```

Comment:
Just wonder if samplePercent can be 0. If so, we may have a core when dividing by sampleRatio.

Reply:
I tried this, actually. We get "infinity" for the floating point number. That's why I added the "isnormal" test. We also error out with a 9207 error if the sample table is empty.